com.avaloq.tools.ddk.xtext.export/src/com/avaloq/tools/ddk/xtext/export/generator/ResourceDescriptionStrategyGenerator.xtend

```diff
@@ -108,8 +108,13 @@ class ResourceDescriptionStrategyGenerator {
                   return «p.name»ExportSwitch.doSwitch(object);
                 }
               «ENDFOR»
+              «IF it.extension»
+              // Extension does not have to cover all EPackages of the language
+              return false;

```

Comment:
That maybe also means that it wouldn't work if an object handled by one extension is nested inside an object handled by another extension.

Reply:
This is true. Will have to revisit that. Should work for the Language example and for the Script Package case Ruben works on. So could be a separate change to revisit this. Sill part of this issue.